cases/widgets.py

```
@@ -153,3 +153,47 @@ def to_python(self, value):
+        self.button_label = button_label
+        super().__init__(*args, **kwargs)
+        self.widget = MultiURLWidget(button_label=button_label)
+    
+    def to_python(self, value):
+        if value in self.empty_values:
+            return []
+        if isinstance(value, list):
+            return value
+        try:
+            return json.loads(value) if value else []
+        except:
```

Ruff (0.15.2)

[error] Do not use bare `except`

(E722)